src/main/java/io/cryostat/net/web/http/api/v2/RequestParameters.java

```diff
@@ -94,6 +94,7 @@ public RequestParameters(
     public static RequestParameters from(RoutingContext ctx) {
         Objects.requireNonNull(ctx, "ctx");
 
+        // TODO: What if null and endpoint requires specific content-type during handle?
```

Comment:
Any ideas here? My initial impression is that any such endpoint handler should just check for `null` and assume some default content-type. The alternative would be to simply respond with a `406`, which could be done in the `Abstract` handler base classes, or perhaps even a new generic/global handler with a lower-than-default priority number, before it gets to the specific endpoint implementation.

Reply:
I guess just basic null-checking makes the most sense since this is the only endpoint for now that deals with multiple values in `produces`. The reason I put that there was because I realized the `fireDownloadRequest` in the `StandardSelfTest` itesting class seemed to not send any type of Accept header at all (not even something like `*/*`) and that was getting an error, so I was confused on to what should happen.